tests/unittests/config/test_schema.py

```diff
@@ -382,8 +382,8 @@ def test_network_config_schema_validation_false_when_skipped(
                     column=12,
                     message="incorrect YAML value: yes for dhcp value",
                 ),
-                "Invalid network-config provided:\nformat-l1.c12: Invalid"
```

Comment:
The point of this test was also to establish that our error message reports the right line/format prefix.
What do you think about the following instead given the whitespace issue?

```diff
--- a/tests/unittests/config/test_schema.py
+++ b/tests/unittests/config/test_schema.py
@@ -382,7 +382,7 @@ class TestNetplanValidateNetworkSchema:
                     column=12,
                     message="incorrect YAML value: yes for dhcp value",
                 ),
-                "Invalid network-config provided:\nformat-l1.c12: Invalid"
+                r"Invalid network-config provided:.*format-l1.c12: Invalid"
                 " netplan schema. incorrect YAML value: yes for dhcp value",
             ),
         ),
@@ -412,9 +412,10 @@ class TestNetplanValidateNetworkSchema:
                 "cloudinit.config.schema.mkdtemp",
                 return_value=fake_tmpdir.strpath,
             ):
-                assert netplan_validate_network_schema({"version": 2})
+                with caplog.at_level(logging.WARNING):
+                    assert netplan_validate_network_schema({"version": 2})
             if error_log:
-                assert error_log in caplog.text
+                assert re.match(error_log, caplog.records[0].msg, re.DOTALL)
 
 
 class TestValidateCloudConfigSchema:
```

Reply:
Sounds good, and thanks for the patch. I verified that the test passes on Focal